tests/registry_disk_test.go

```diff
@@ -37,7 +37,7 @@ import (
 	"kubevirt.io/kubevirt/tests"
 )
 
-var _ = Describe("ContainerDisk", func() {
+var _ = Describe("[rfe_id:588, crit:medium, vendor:cnv-qe@redhat.com, level:component]ContainerDisk", func() {
```

Comment:
please update tags to the format [...][...]

Reply:
yes, i thought i had already fixed this, but just realized it was for kubevirt-ansible PR of mine. Fixing it now.